src/darwin/CHIPTool/CHIPTool/QRCode/QRCodeViewController.m

```diff
@@ -233,9 +233,9 @@ - (void)handleRendezVous:(CHIPSetupPayload *)payload
     case kRendezvousInformationAllMask:
         NSLog(@"Rendezvous Unknown");
         break;
-    case kRendezvousInformationSoftAP:
-        NSLog(@"Rendezvous SoftAP");
-        [self handleRendezVousSoftAP:[self getNetworkName:payload.discriminator]];
+    case kRendezvousInformationWiFi:
+        NSLog(@"Rendezvous Wi-Fi");
```

Comment:
nit: `WiFi` instead of `Wi-Fi`

Reply:
Pedantically, the official Wi-Fi Alliance spelling is "Wi-Fi" FWIW. I think we should hew to the official spelling.

Reply:
I thought I recalled a discussion in a previous PR discussing `WiFi` vs `Wi-Fi` and my memory was that we had landed on `WiFi`, however I can't seem to locate said PR/conversation. To be honest I don't have a strong preference as long as we are consistent :)